web/components/ScheduleHeader.jsx

```diff
@@ -1,12 +1,35 @@
 import React from 'react';
+import moment from 'moment';
+
+import Text from './Text';
+import Card from './Card';
+import Flex from './Flex';
+
 
 export default class ScheduleHeader extends React.Component {
     render() {
+        const { selectedDay } = this.props;
         return (
-            <div className='ScheduleHeader'>
-                <h6>ScheduleHeader</h6>
-            </div>
+            <Flex
+                row
+                className='ScheduleHeader'
+            >
+                <Card
+                    width='10px'
+                    height='10px'
+                    bg='green' // TODO This color must be the color of the member
```

Comment:
Please track this as a separate issue for this TODO and remove the code comment

Reply:
On second thought, let's actually keep the ScheduleHeader as a fixed color. It's a nice to have but could have a negative impact on UI if there are too many different random colors. No need to worry about color scheming just yet